### PR TITLE
cli: add unit for generateTerraformVariablesStep

### PR DESCRIPTION
**Pull request by enxebre**
Add unit for generateTerraformVariablesStep

### CODE REVIEW

**Comment by squat** [comment]
let's wait until https://github.com/coreos/tectonic-installer/pull/3046 merges

**Comment by enxebre** [comment]
@squat Rebased and updated

File: installer/pkg/workflow/init_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	terraformVariablesFilePath := filepath.Join(m.clusterDir, terraformVariablesFileName)
+	gotData, err := ioutil.ReadFile(terraformVariablesFilePath)
+	if err != nil {
+		t.Errorf("Test case TestGenerateTerraformVariablesStep: failed to ReadFile(): %s", err)
+	}
+	got := string(gotData)
+
+	expectedData, err := ioutil.ReadFile("./fixtures/terraform.tfvars")
+	if err != nil {
+		t.Errorf("Test case TestGenerateTerraformVariablesStep: failed to ReadFile(): %s", err)
```
Comment (squat) [inline installer/pkg/workflow/init_test.go]: Both of these errors are the same. Maybe you can specify what you are loading: “failed to load tfvars”

File: installer/pkg/workflow/init_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+func initCluster(t *testing.T, file string) config.Cluster {
+	testConfig, err := config.ParseFile(file)
+	if err != nil {
+		t.Errorf("Test case TestGenerateTerraformVariablesStep: failed to parse test config, %s", err)
+	}
+	return testConfig.Clusters[0]
+}
+
+func TestGenerateTerraformVariablesStep(t *testing.T) {
+
```
Comment (squat) [inline installer/pkg/workflow/init_test.go]: Extra newline here

File: installer/pkg/workflow/init_test.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	cluster := initCluster(t, "./fixtures/aws.basic.yaml")
+	clusterDir := "./"
```
Comment (squat) [inline installer/pkg/workflow/init_test.go]: Conventionally we use “.”

File: installer/pkg/workflow/init_test.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/coreos/tectonic-installer/installer/pkg/config"
+)
+
+func initCluster(t *testing.T, file string) config.Cluster {
```
Comment (squat) [inline installer/pkg/workflow/init_test.go]: Is this name too generic that the will collide with something in the actual package in the future?

**Comment by enxebre** [comment]
@squat many thanks for the feedback, made some changes according to it

**Pull request merged**